### DISCUSSION TITLE
Moving the < >

### DISCUSSION THREAD

**Discussion by ellis246** [nolimits4web/swiper#6298] [Q&A]
Quite a simple question, apologies if it's too simple but I couldn't see how to do this. At least not built into the API.

How would I move the < and > arrows to the top of the page, rather than how they sit by default midway on the page?

Thanks

**Comment by onezeno717** [comment]
You can do it manually by writing its classes. https://swiperjs.com/swiper-api#navigation
for example :

`const swiper = new Swiper('.swiper', {
  navigation: {
    nextEl: '.my-suctom-swiper-button-next',
    prevEl: '.my-suctom-swiper-button-prev',
  },
});`
